src/ViURShopClient.js

```diff
@@ -422,6 +422,18 @@ export class ViURShopClient {
             .then(req => req.json());
     }
 
+    discount_remove({
```

Comment:
Can you please add a JS doc string to the method? we should add it to every method we touch, otherwise it will never be documented.
**Tip:** Write the docstring with parameter descriptions once and use the same in the python backend API method and here in the client. :)